tools/cibsecret.in

```diff
@@ -339,6 +347,10 @@ localfiles() {
 cibsecret_set() {
     CS_VALUE="$1"
 
+    while [ -z "$CS_VALUE" ]; do
+        read -p "Enter value: " CS_VALUE
+    done
+
```

Comment:
It's legal to set a parameter to the empty string, though I'm not sure I've seen it in the wild. Let's check the initial number of arguments instead of whether the string is empty, and always take the first entered value -- they can control-C if they change their mind.

Reply:
Done.